db/patterns/nespresso.eno

```diff
@@ -0,0 +1,12 @@
+name: Nespresso
+category: site_analytics
+website_url: https://www.nespresso.com/
+organization: nestle
+
+--- domains
+nespresso.com
```

Comment:
```suggestion
servertag.nespresso.com
```

That could make it more precise that we are targeting the analytics component. Otherwise, it feels very broad; a bit like putting Coca Cola as an analytics framework.